### DISCUSSION TITLE
Mostly used language in GitHub

### DISCUSSION THREAD

**Discussion by shree-2802** [anuraghazra/github-readme-stats#3595] [Q&A] [answered]
Hi all, I have a problem with most commonly used language card in github, though i have uploaded many other files of other languages, only javascript,css and html is showing in the card.. What should I do?

<p><img align="left" src="https://github-readme-stats.vercel.app/api/top-langs?username=shree-2802&show_icons=true&locale=en&layout=compact" alt="shree-2802" /></p> 

"https://github-readme-stats.vercel.app/api/top-langs?username=shree-2802&show_icons=true&locale=en&layout=compact"

This is the link which i used.. But i also used other links no use but then..

**Comment by qwerty541** [comment, marked as answer]
Hey, @shree-2802! Currently your card shows Typescript and SCSS as well, so I assume that the problem can be connected to caching. Be aware that our cards have default cache of 6 hours. If you want the card to be updated more often, then you can [deploy own Vercel instance](https://github.com/anuraghazra/github-readme-stats#on-vercel) and set [environment variable](https://github.com/anuraghazra/github-readme-stats#disable-rate-limit-protections) `CACHE_SECONDS`.
